frontend/package.json

```diff
@@ -25,7 +25,7 @@
 		"docs": "typedoc --out ./docs ./src/globals --options ./internals/typedoc.js --logger none",
 		"extract": "node ./internals/intl/extract.js",
 		"compile": "node ./internals/intl/compile.js",
-		"storybook": "start-storybook -p 6006",
+		"storybook": "start-storybook -p 6006 -s ../public",
```

Comment:
Why did this was changed?

Reply:
It was to load the default cover image (the 3d repo blue label on white bg) using the same path as in the rest of the code.
I will remove it as the new changes make this futile.